### ISSUE TITLE
Panic due to nil pointer dereference in sechub

### ISSUE THREAD

**Issue by htnosm** [pepabo/control-controls#39]
### Version

v0.8.1

### Description

The following error is occurring:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9855fe]

goroutine 1 [running]:
github.com/pepabo/control-controls/sechub.ctrls({0xc35f50, 0xc000092000}, 0xc000092000?, 0xc0002e6c20)
	/Users/runner/work/control-controls/control-controls/sechub/sechub.go:384 +0x33e
github.com/pepabo/control-controls/sechub.(*SecHub).Fetch(_, {_, _}, {{0xc00015a5e0, 0xa}, {0xc30120, 0xc0000a3240}, {0x0, 0x0}, {0x0, ...}, ...})
	/Users/runner/work/control-controls/control-controls/sechub/fetch.go:28 +0x1de
github.com/pepabo/control-controls/sechub.(*SecHub).Plan(_, {_, _}, {{0xc00015a5e0, 0xa}, {0xc30120, 0xc0000a3240}, {0x0, 0x0}, {0x0, ...}, ...}, ...)
	/Users/runner/work/control-controls/control-controls/sechub/plan.go:45 +0x248
github.com/pepabo/control-controls/cmd.glob..func4(0xfbe800?, {0xc000147290, 0x1, 0x3?})
	/Users/runner/work/control-controls/control-controls/cmd/plan.go:67 +0x9d5
github.com/spf13/cobra.(*Command).execute(0xfbe800, {0xc000147200, 0x3, 0x3})
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xfbeae0)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/pepabo/control-controls/cmd.Execute()
	/Users/runner/work/control-controls/control-controls/cmd/root.go:61 +0x256
main.main()
	/Users/runner/work/control-controls/control-controls/main.go:27 +0x17
```

Although I am not sure about the exact occurrence date, it was functioning normally until around the end of May 2024.
It seems that the error occurs when `DescribeStandardsControls` is executed and `DisabledReason` is not present.
Checking the return value and inserting an empty string when it is nil seems to resolve the issue.
However, The impact on other parts is unknown.

**Comment by k1LoW** [comment]
ref: https://github.com/pepabo/control-controls/pull/40

**Comment by k1LoW** [comment]
@htnosm Sorry for reply 💦 

Fixed as v0.8.3.

**Comment by htnosm** [comment]
@k1LoW  The issue was resolved by upgrading the version. Thank you for your assistance.
